### PR TITLE
Cleans up living duffel bag code to meet guidelines

### PR DESCRIPTION
**Pull request by necromanceranne**
## About The Pull Request

Cleans up some of the living duffel bag code a little more. Removes a hard deletion as well as makes some variables meet contribution guidelines.

## Why It's Good For The Game

I SPIL MY CDE GRENS HELP HELP HELP HELP HELP

## Changelog
:cl:
code: Tidies up some of the code in living duffel bags to meet contribution guidelines. 
/:cl:

### CODE REVIEW

**Review by Rohesie — APPROVED** [review]

**Comment by Rohesie** [comment]
Since you're here you can go further and clean the rest of the file.
Remove the `src` part of stuff like `src.loc` for example as well.

**Comment by necromanceranne** [comment]
> Since you're here you can go further and clean the rest of the file.
> Remove the `src` part of stuff like `src.loc` for example as well.

donezo

**Review by Rohesie — APPROVED** [review]

File: code/game/objects/items/storage/backpack.dm [hunk 1]
```diff
@@ -400,16 +398,16 @@
 	if((hunger > 50) && prob(20))
 		for(var/obj/item/I in contents)
 			if(IS_EDIBLE(I))
-				var/obj/item/food/F = I
-				F.forceMove(user.loc)
+				var/obj/item/food/hunger_breaks = I //If you fed them poundland microwave meals, it probably would kill them
```
Comment (Mothblocks) [inline code/game/objects/items/storage/backpack.dm:401]: I guess this is out of scope for this PR, but this code is wrong. It implies everything that `IS_EDIBLE` passes for is `/obj/item/food`, which isn't true.

Comment (necromanceranne) [reply code/game/objects/items/storage/backpack.dm:401]: Oh, uh, that's probably quite true I would have fixed it up.

**Pull request merged**